DigitalLearningSolutions.Web/Services/RegistrationService.cs

```diff
@@ -349,7 +378,19 @@ public void CreateCentreManagerForExistingUser(int userId, int centreId, string?
                 userAccount.ProfileImage
             );
 
-            registrationDataService.RegisterAdmin(new AdminAccountRegistrationModel(registrationModel, userId));
+            registrationDataService.RegisterAdmin(
+                new AdminAccountRegistrationModel(registrationModel, userId),
+                new PossibleEmailUpdate
+                {
+                    OldEmail = userDataService.GetCentreEmail(userId, centreId),
+                    NewEmail = centreSpecificEmail,
+                    NewEmailIsVerified = emailVerificationDataService.AccountEmailIsVerifiedForUser(
+                                             userId,
+                                             centreSpecificEmail
+                                         ),

```

Comment:
This whitespace is wrong (if I save the file the closing bracket moves to inline with the N of `NewEmailIsVerified`